packages/immutable-rootfs/30cos-immutable-rootfs/cos-loop-img.sh

```diff
@@ -9,16 +9,38 @@ function doLoopMount {
         label=$(basename "${dev}")
         [ -e "/tmp/cosloop-${label}" ] && continue
         > "/tmp/cosloop-${label}" 
+
         mount -t auto -o "${cos_root_perm}" "/dev/disk/by-label/${label}" "${cos_state}" || continue
         if [ -f "${cos_state}/${cos_img}" ]; then
-            losetup -f "${cos_state}/${cos_img}"
+
+            # FSCHECK if cos_root_perm == "ro" on both
+            if [ "$cos_root_perm" == "ro" ]; then
+               /usr/lib/systemd/systemd-fsck "/dev/disk/by-label/${label}"
```

Comment:
This absolute path frightens me... I am not sure if we have the `${systemdutilsdir}` variable here, I doubt it though. Probably we could install a symlink under `/sbin` in module_setup.sh and just use `systemd-fsck` here?

Reply:
I know, it frightens me as well, but the fact that systemd-fsck is not having a binary in /sbin etc hold me off from doing any special setup. I just wonder why is not installed as a normal binary to begin with ?!

Reply:
Seems it is? https://github.com/dracutdevs/dracut/blob/d502d2a816ba8f8329b3d8616bd2a7e82a0ad21f/modules.d/00systemd/module-setup.sh#L39

Reply:
mmh but might not be just available when calling it inside scripts. I just wonder how dracut handles it as it used already?

Reply:
should be fixed now!